ui/shared/forms/FileSnippet.tsx

```diff
@@ -76,7 +76,7 @@ const FileSnippet = ({ file, className, index, onRemove, isDisabled, error }: Pr
             <Tooltip
               label={ error }
               placement="top"
-              maxW="320px"
+              maxW={{ base: 'calc(100vw - 8px)', lg: '320px' }}
```

Comment:
If `calc(100vw - 8px)` is the default value, then here and in other places we can just set a value for desktop viewports:

```suggestion
              maxW={{ lg: '320px' }}
```

Reply:
No, this will overwrite the default value